fvid/fvid.py

```diff
@@ -200,6 +200,7 @@ def main():
 
     parser.add_argument("-i", "--input", help="input file", required=True)
     parser.add_argument("-o", "--output", help="output path")
+    parser.add_argument("-f", "--framerate", help="set framerate for encoding (as a fraction)", action='store_const', const="1/5", type=str)
```

Comment:
There is an issue here, 

`action='store_const', const="1/5"`

from testing that throws a

`TypeError: __init__() got an unexpected keyword argument 'type'`

I think it because **const** cannot be string. Looking at the [argparse documentation](https://docs.python.org/3/library/argparse.html#action), in their examples, they use integers when using action="store_const"

This is a possible fix that worked for me:
`     parser.add_argument("-f", "--framerate", help="set framerate for encoding (as a fraction)", default="1/5", type=str)`

Other than that, it looks great! I really like the way you're checking for negative fractions, I haven't see that before.

Reply:
Should be fixed, will leave to you to resolve if you agree it's fixed.